client/src/redux/AuthRedux.test.js

```diff
@@ -20,7 +20,8 @@ import Actions, { reducer, INITIAL_STATE } from './AuthRedux';
 test('loginRequest', () => {
```

Comment:
There shouldn't be any change in `next-logo-primary.png`. Only files that have changed should appear in the PR.